compose-builder/add-taf-app-services.yml

```diff
@@ -42,15 +42,15 @@ services:
     image: ${APP_SVC_REPOSITORY}/app-service-configurable${ARCH}:${APP_SERVICE_VERSION}${APP_SVC_DEV}
     ports:
       - "59710:59703" #Exposing as different port to avoid conflict with other MQTT export instance
-    container_name: scalability-test-mqtt-export
-    hostname: scalability-test-mqtt-export
+    container_name: edgex-scalability-test-mqtt-export
+    hostname: edgex-scalability-test-mqtt-export
```

Comment:
use edgex-taf-scalability-test-mqtt-export?

Reply:
No the service key is `scalability-test-mqtt-export`